lib/vanagon/platform/dsl.rb

```diff
@@ -38,7 +38,7 @@ def platform(platform_name, override_name: false, &block)
         @platform = case platform_name
                     when /^aix-/
                       Vanagon::Platform::RPM::AIX.new(@name)
-                    when /^(cisco-wrlinux|el|fedora|redhat|redhatfips)-/
+                    when /^(cisco-wrlinux|el|fedora|redhat|redhatfips|amazon)-/
```

Comment:
Optional small suggestion, might be worth sorting this alphabetically 
```suggestion
                    when /^(amazon|cisco-wrlinux|el|fedora|redhat|redhatfips)-/
```